reference/command/credentials.md

```diff
@@ -46,21 +46,23 @@ Signature:  b235429f8dc7be2e79bca0b8f59bdb6676b06f608408085097e7fb5a2029de0d27d6
 ### Verifying Credentials
 
 ```
+» ockam reset -y
```

Comment:
We don't want people to reset after they have enrolled, `ockam delete node --all` where possible.

Reply:
It don't work this way because:
1. Nodes act differently if enrolled
2. Identities are not deleted with ockam node delete